dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/master/processor/queue/TaskResponseService.java

```diff
@@ -142,84 +175,72 @@ public void run() {
     }
 
     /**
-     * persist  taskResponseEvent
-     *
-     * @param taskResponseEvent taskResponseEvent
+     * event handler thread
      */
-    private void persist(TaskResponseEvent taskResponseEvent) {
-        Event event = taskResponseEvent.getEvent();
-        Channel channel = taskResponseEvent.getChannel();
+    class EventHandler extends Thread {
```

Comment:
name can be more concrete like `TaskResponseEventHandler`

and the comment `event handler thread` say nothing